beetmoverscript/templates/maven_components.yml

```diff
@@ -0,0 +1,50 @@
+---
+metadata:
+    name: "Maven repository"
+    description: "Maps artifacts to spec'd maven location"
+    owner: "release@mozilla.com"
+
+s3_bucket_path: maven2/org/mozilla/components/{{ artifact_id }}/{{ version }}/
+
+mapping:
+{% for locale in ['en-US'] %}
+  "{{ locale }}":   # Locale is not needed for components, it's used by move_beets, though
+  {% for product in ['components'] %}
```

Comment:
Nit: Is `product` used in this loop? If I remember correctly I made the same nit in maven_geckoview.yml.

Reply:
Good catch. Slippery copy-paste :) Want me to remove it from `maven-geckoview.yml` as well?

Reply:
Yes, please :)